docs/product-getting-started-guide/src/main/asciidoc/business-rules-create-proc.adoc

```diff
@@ -0,0 +1,73 @@
+[id='guided-rule-licence-suspension-create-proc']
+= Creating the Driver License Suspension rule
+
+The following chapter instructs you how to create a driver license suspension rule in {PRODUCT} using the Guided Rule wizard.
+
+[float]
+== Prerequisite
+
+Created both the Violation and Driver data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Guided Rule*, then enter:
+
+ * *Guided Rule*: `DriverLicenseSuspensionRule`
+ * *Package*: `org.jboss.example.traffic_violations`
++
+
+. Click *+Ok* to open the *Guided Rule designer*.
++
+
+.Create new Guided Rule window
+image::guided-rule.png[]
+
+== Setting the Suspension rule conditions
+
+. Click image:6176.png[] next to the *WHEN* label to open the *Add a condition to the rule* window. . . Select *Driver...* and click *Ok*.
++
+
+.Create new Guided Rule window
+image::add-condition.png[]
+. Click the *There is a Driver* label to open the *Modify constraints for Driver* window.
+. Enter `driver` in the *Variable name* field and click *Set*.
+. Click the *There is a Driver[driver] with* and click *Expression editor*
+. Click *[not bound]* to open the *Expression editor*.
+. In the *Bind the Expression to a new variable* field, enter: `previousPts` and click *Set*.
+. From the *Choose...* menu, select *totalPoints*.
+. Click image:6176.png[] next to line 1 (the *previousPts* label) to open the *Add a condition to the rule* window.
+. Select *From Accumulate...* and click *+Ok*.
+. Select *click to add pattern...* above the *From Accumulate* label and select *Number* from the *choose fact type* pull-down menu.
+. Click *There is a number* label to open the *Modify contraints for Driver* window.
+. Click *Expression editor* and select *intValue()* from the
+*[not bound]:* *Choose...* menu.
+. Click *[not bound]* to open the *Expression editor*.
+. In the *Bind the Expression to a new variable* field, enter: `totalNewPoints` and click *Set*.
+. Select *click to add pattern* below the *From Accumulate...* label and select *Violation* from the *choose fact type* pull-down menu.
+. Click *All Violation with:* to open the *Modify contraints for Violation* window and select *points* from the *Add a restriction on a field* pull-down menu.
+. Click *--- please choose ---* next to the *points* label and select *greater than*.
+. Click image:6191.png[], then click *Literal value*.
+. Click the *points* label to open the *Add a field* window and input `vPoints` and click *Set*.
+. In the *Function* field, enter `sum(vPoints)`.
+. Select *greater than or equal to* from the *totalNewPoints* -> *please choose* pull-down menu.
+. Click image:6176.png[] next to *greater than or equal to*, click *New formula*, and enter `20-previousPts` in the new field.
+. Click *Save*, then click *Save* to confirm your changes.
```

Comment:
Here and elsewhere in the doc. I know you probably don't care about this at this point, and I probably wouldn't either, so maybe just for future reference: Procedures typically shouldn't have more than like a dozen or at most 15 steps. IBM SG has some number that I don't recall, but point being, any more than that, and procedure should and probably can be made into separate sections/tasks. This is an edge case and is only that many steps because of how granular this is (as a tutorial), so may be an exception. But just wanted to mention it.

Reply:
I spoke with my SME (Ivo) about how to logically break the steps up. We did the best we could without losing the contain logic. It bothers me to.